example/lib/src/screen.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:flutter/material.dart';
+import '../utils/routes_constants.dart';
+import '../utils/strings_constants.dart';
+import '../utils/text_styles.dart';
+import '../utils/constants.dart';
+
+class Screen extends StatelessWidget {
```

Comment:
Can you change this class name to another more descriptive name?

Reply:
done